types/storybook__addon-knobs/index.d.ts

```diff
@@ -56,3 +56,7 @@ export interface WrapStoryProps {
 
 export function withKnobs(storyFn: RenderFunction, context: StoryContext): React.ReactElement<WrapStoryProps>;
 export function withKnobsOptions(options: { debounce: boolean, timestamps: boolean }): (storyFn: RenderFunction, context: StoryContext) => React.ReactElement<WrapStoryProps>;
+
+/// <reference path="./angular.d.ts" />
```

Comment:
These shouldn't be necessary if these files are re-written to avoid `declare module` (see above comment)

Reply:
a65fd408d25c20942f489d136079bef4ba879518